docs/tools/terraform/get-started.rst

```diff
@@ -1,57 +1,56 @@
-Set up your first Aiven Terraform project
-=========================================
+Get started with Aiven Provider for Terraform
+==============================================
 
-This example shows the setup for a Terraform project containing a single Redis®* service, and shows off some useful commands to stand up (and destroy) your Aiven data infrastructure.
+This example shows you how to use the Aiven Provider to set up your data infrastructure by creating a single Aiven for Redis®* service in an :doc:`Aiven project </docs/platform/concepts/projects_accounts_access>`. 
 
-Prepare the dependencies 
-''''''''''''''''''''''''
-- `Download and install Terraform <https://www.terraform.io/downloads>`_
-- `Sign up <https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo>`_ for Aiven if you haven't already
-- `Generate an authentication token <https://docs.aiven.io/docs/platform/howto/create_authentication_token.html>`_
+.. caution::
+
+  Recreating stateful services with Terraform may delete the service and all its data before creating it again. Some properties, like project and resource name, cannot be changed and it will trigger a resource replacement. Check the Terraform plan to find out whether a service will be deleted or replaced.
```

Comment:
nit: terraform plan _command_? I mean this absolutely correct sentence, but late it explains for the very first time what plan is. And the word is confusing, like if terraform usage has own "price plan". Adding a link to the below example might also help the understanding what plan is.